plugins/commander.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: commander
+spec:
+  homepage: https://github.com/schabrolles/kubectl-commander.git
+  shortDescription: Interactively manage Kubernetes resources.
+  version: "v0.4.0"
+  description: |
+    kube-commander is an interactive menu (based on fzf) that allows you
+    to navigate and interact with objects of your Kubernetes cluster.
+    It is possible to quickly preview object configuration in YAML,
+    logs, events but also edite or delete object or enter into pods.
+
+    Help:
+    $ kubectl commander -h
+    [ctrl-h] inside the plugin
```

Comment:
If I recall correctly, krew prints this after installing every plugin so it is redundant.